chapters/synchronous.tex

```diff
@@ -1400,13 +1400,13 @@ \subsection{Inferencing of solverMethod}\label{inferencing-of-solvermethod}
 
 \section{Initialization of Clocked Partitions}\label{initialization-of-clocked-partitions}
 
-The standard scheme for initialization of Modelica models does not apply for clocked base-partitions.
+The standard scheme for initialization of Modelica models does not apply for discrete-time sub-partitions.
```

Comment:
Didn't we drop the _sub_ here?  I can't find a single example of _discrete-time sub-partition_ in the current text?
```suggestion
The standard scheme for initialization of Modelica models does not apply for discrete-time partitions.
```